flink-connectors/flink-connector-kinesis/pom.xml

```diff
@@ -35,7 +35,7 @@ under the License.
 	<properties>
 		<aws.sdk.version>1.11.603</aws.sdk.version>
 		<aws.kinesis-kcl.version>1.11.2</aws.kinesis-kcl.version>
-		<aws.kinesis-kpl.version>0.13.1</aws.kinesis-kpl.version>
+		<aws.kinesis-kpl.version>0.14.0</aws.kinesis-kpl.version>
```

Comment:
Can you also update the version in `flink.git/flink-connectors/flink-connector-kinesis/src/main/resources/META-INF/NOTICE`, as well as potential transitive version changes. (The maven-shade-plugin will show you the included artifacts + their versions)